README.md

```diff
@@ -17,54 +17,75 @@ Features:
 npm install uuid
 ```
 
-```javascript
-// Generate a v1 UUID (time-based)
-const uuidV1 = require('uuid/v1');
-uuidV1(); // -> '6c84fb90-12c4-11e1-840d-7b25c5ee775a'
+Then generate your uuid version of choice ...
 
+Version 1 (timestamp):
 
-// Generate a v4 UUID (random)
-const uuidV4 = require('uuid/v4');
-uuidV4(); // -> '110ec58a-a0f2-4ac4-8393-c866d813b8d1'
+    const uuidv1 = require('uuid/v1');
+    uuidv1(); // -> '6c84fb90-12c4-11e1-840d-7b25c5ee775a'
 
+Version 4 (random):
 
-// Generate a v5 UUID (namespace)
-const uuidV5 = require('uuid/v5');
+    const uuidv4 = require('uuid/v4');
```

Comment:
would be nice to be consistent with the code examples below and the the triple backtick syntax